### DISCUSSION TITLE
[@types/three] WebGLRenderer.outputColorSpace missing

### DISCUSSION THREAD

**Discussion by rediche** [DefinitelyTyped/DefinitelyTyped#65326] [Issues with a @types package] [answered]
In version 152 of ThreeJS `WebGLRenderer.outputEncoding` was replaced with `WebGLRenderer.outputColorSpace`.

Source: https://github.com/mrdoob/three.js/wiki/Migration-Guide#151--152

[a comment by an automated account is not shown]

**Comment by Methuselah96** [comment, marked as answer]
Thanks for the discussion. We're just finishing up getting the changes for r151 released in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65311. The changes for r152 should hopefully be released in the next week or two and it looks like https://github.com/three-types/three-ts-types/pull/397 already contains these changes, so we should be good to go. Sorry for the delay.

**Reply by rediche** [reply]
Absolutely no problem. It was just a coincidence that we had an upgrade planned and it ended up being straight after the Three r152-release. 😅

Thanks for the answer, looking forward to it.

**Reply by Methuselah96** [reply]
It should be released soon (https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65356), just waiting on a DT maintainer review.
